ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.html

```diff
@@ -1,6 +1,15 @@
 <app-container>
   <h1 class='theme-primary-fg' i18n>Processing Tasks</h1>
   <div class='controls'>
+    <div *ngIf='roles._.pipelinesNew'>
```

Comment:
The roles object details what the user is allowed to do.
We can allow all users to sort the pipelines, so there's no need in the `*ngIf` here